omise-woocommerce.php

```diff
@@ -111,7 +137,7 @@ public function load_plugin_textdomain() {
 	public function register_user_agent() {
 		global $wp_version;
 
-		$user_agent = sprintf( 'OmiseWooCommerce/%s WordPress/%s WooCommerce/%s', OMISE_WOOCOMMERCE_PLUGIN_VERSION, $wp_version, WC()->version );
+		$user_agent = sprintf( 'OmiseWooCommerce/%s WordPress/%s WooCommerce/%s', OMISE_WOOCOMMERCE_PLUGIN_VERSION, $wp_version, class_exists('WC') ? WC()->version : '' );
```

Comment:
that function is called anyway.
because plugin is loaded, so all plugins_loaded actions are triggered.
We need to check than if WC object is defined (woocommerce is active)

Reply:
👍 ic